openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -175,95 +175,102 @@ The following subsections illustrate some of the authorization flows supported b
 
 ## Authorization Code Flow {#authorization-code-flow}
 
-Below is a diagram of a Credential issuance using the Authorization Code Flow, using grant type `authorization_code` as defined in [@!RFC6749].
+The Authorization Code Flow uses the grant type `authorization_code` as defined in [@!RFC6749] to issue Credentials.
```

Comment:
```suggestion
The Authorization Code Flow uses the grant type `authorization_code` as defined in [@!RFC6749] to issue the Access Token required to issue the Credentials at the Credential endpoint defined in this document.
```

Since the auth code flow defined RFC6749 is used to issue the Access Tokens used to protect the Credential endpoint.

or, we may say 

````
The Authorization Code Flow uses the `authorization_code` grant type, as specified in [@!RFC6749], to generate the Access Token necessary to issue the Credentials at the Credential endpoint, as outlined in this document.
````

Reply:
I would keep this rather short:
```suggestion
The Authorization Code Flow uses the grant type `authorization_code` as defined in [@!RFC6749] to issue Access Tokens.
```